app/src/main/java/org/wikipedia/feed/becauseyouread/BecauseYouReadCardView.java

```diff
@@ -27,21 +27,21 @@ public BecauseYouReadCardView(Context context) {
     }
 
     private void header(@NonNull final BecauseYouReadCard card) {
-        headerView().setTitle(card.title())
+        getHeaderView().setTitle(card.title())
```

Comment:
Since it is still a Java class, it will be using `getHeaderView().` instead of `headerView.`. Once we convert this to Kotlin file it will be resolved.